test_pytest_rerunfailures.py

```diff
@@ -237,13 +238,17 @@ def test_fail():
     result = testdir.runpytest('--reruns', '3',
                                '--reruns-delay', str(delay_time))
 
+    num_warnings = 0
```

Comment:
This could probably be a `has_warnings` or `warning_expected` boolean now?

Reply:
ah yes, this is wrong. sorry for leaving it, let me remove it

Reply:
The new version looks better, though now you have two identical `if` statements after each other.

Reply:
sorry just changed it